ARX/OCVT/PlanarOrbTracker.cpp

```diff
@@ -446,24 +431,24 @@ class PlanarOrbTracker::PlanarOrbTrackerImpl
 
     void CameraPoseFromPoints(cv::Mat pose, std::vector<cv::Point3f> oPts, std::vector<cv::Point2f> iPts)
     {
-        //cv::Mat rvec = cv::Mat::zeros(3, 1, CV_64FC1);          // output rotation vector
-        //cv::Mat tvec = cv::Mat::zeros(3, 1, CV_64FC1);          // output translation vector
-        cv::Mat rvec;          // output rotation vector
-        cv::Mat tvec;          // output translation vector
+        cv::Mat rvec = cv::Mat::zeros(3, 1, CV_64FC1);          // output rotation vector
+        cv::Mat tvec = cv::Mat::zeros(3, 1, CV_64FC1);          // output translation vector
+        //cv::Mat rvec;          // output rotation vector
+        //cv::Mat tvec;          // output translation vector
         std::cout << "Inside cameraPose\n" << std::endl;
         // --llvm-lto 1 compiler setting breaks the solvePnPRansac function on iOS but using the solvePnP function is faster anyways
         #if ARX_TARGET_PLATFORM_EMSCRIPTEN
           std::cout << "before solvePnP\n" << std::endl;
-          bool solvePnP = cv::solvePnP(oPts, iPts, _K, NULL, rvec, tvec);
+          bool solvePnP = cv::solvePnP(oPts, iPts, _K, cv::noArray(), rvec, tvec);
```

Comment:
this solved the pose issue (stack oveflow error)!! 
using `cv::noArray()` instead of `NULL`